src/Cli/Utils.cs

```diff
@@ -840,5 +840,15 @@ public static bool IsEntityProvided(string? entity, ILogger cliLogger, string co
 
             return true;
         }
+
+        /// <summary>
+        /// Returns ILoggerFactory with CLI custom logger provider.
+        /// </summary>
+        public static ILoggerFactory GetLoggerFactoryForCli()
```

Comment:
should callers be asking for a logger factory or a logger? ILoggerFactory has a method called `CreateLogger(string categoryName)` which returns a logger.